android/build.gradle

```diff
@@ -144,7 +144,6 @@ allprojects {
 }
 
 ext {
-    buildToolsVersion = "26.0.2"
```

Comment:
Is this change intentional?

Reply:
Yes, the new version doesn't care about it and uses its default one

Reply:
Let's be specific. For reproducible builds. We have had crashes in the past due to the version here.

Reply:
Doh, ignore this, GH wasn't showing me the next line.